### ISSUE TITLE
Loss Function question

### ISSUE THREAD

**Issue by inspirit** [nii-yamagishilab/Capsule-Forensics-v2#3]
Hello,

would you be so kind to explain why you are using Softmax here:
https://github.com/nii-yamagishilab/Capsule-Forensics-v2/blob/f8c55d68c74715b9b85434ec8cbdc016730b8711/model_big.py#L185

and then during loss calculation:
https://github.com/nii-yamagishilab/Capsule-Forensics-v2/blob/f8c55d68c74715b9b85434ec8cbdc016730b8711/model_big.py#L201

you use "nn.CrossEntropyLoss()" which assume that input should be logits (raw, unnormalized scores for each class) not probabilities like you do.

Thank you

**Comment by honghuy127** [comment]
Hi,

There is no big difference between the two cases, however, in some rare situation, it could help improve the results (just a little). You can use it or not without any problem.
